app/forms/validators.py

```diff
@@ -405,13 +406,21 @@ def format_playback_value(value, currency=None):
     return format_number(value)
 
 
+def format_message_with_title(error_message, question_title):
```

Comment:
This is very minor, but I didn't immediately pick up what this method was doing from its name, when I looked at it I realised it was just formatting the error_message, perhaps format_error_message_with_title might help developers like me.

Reply:
It could be, but then there isn't a use case where we need it to be so flexible and would require additional unit tests. I think keeping it with the one kwarg is fine until we have a need for anything more complex.